test/e2e/traces_test.go

```diff
@@ -0,0 +1,245 @@
+			token, -1)
+
+		otelFile, err := ioutil.TempFile(e.SharedDir(), "fwd-coll.yaml")
+		testutil.Ok(t, err)
+
+		defer os.Remove(otelFile.Name())
+
+		_, err = otelFile.Write([]byte(forwardingConfig))
+		testutil.Ok(t, err)
+
+		// otelFile.Name() will give relative pathname to temp file.  Docker will complain with
+		// "If you intended to pass a host directory, use absolute path.""
+		otelFileName, err := filepath.Abs(otelFile.Name())
+		testutil.Ok(t, err)
+
+		otel := e.Runnable("otel-fwd-coll").
+			WithPorts(
+				map[string]int{
+					"http": 4318,
+					"grpc": 4317,
+				}).
+			Init(e2e.StartOptions{
+				// Note that if the forwarding collector was OIDC flow we would need
+				// "otel/opentelemetry-collector-contrib:0.45.0" instead.
+				Image:   "otel/opentelemetry-collector:0.45.0",
+				Volumes: []string{fmt.Sprintf("%s:/conf/collector.yaml", otelFileName)},
+				Command: e2e.Command{
+					Args: []string{"--config=/conf/collector.yaml"},
+				},
+			})
+
+		testutil.Ok(t, e2e.StartAndWaitReady(otel))
+
+		// Send trace insecurly to forwarding OTel collector for forwarding through Observatorium
+		// (This code could be refactored to observatorium/up, the test client)
+
+		client := &http.Client{}
+		request, err := http.NewRequest(
+			"POST",
+			fmt.Sprintf("http://%s/v1/traces", otel.Endpoint("http")),
+			bytes.NewBuffer([]byte(traceJSON)))
+		testutil.Ok(t, err)
+		request.Header.Set("Content-Type", "application/json")
+		response, err := client.Do(request)
+		testutil.Ok(t, err)
+		defer response.Body.Close()
+
+		body, err := ioutil.ReadAll(response.Body)
+		testutil.Ok(t, err)
+
+		bodyStr := string(body)
+		assertResponse(t, bodyStr, "{}")
+
+		testutil.Equals(t, http.StatusOK, response.StatusCode)
+
+		request, err = http.NewRequest(
+			"GET",
+			fmt.Sprintf("http://%s/api/v3/traces/%s", httpQueryEndpoint, "5B8EFFF798038103D269B633813FC60C"),
+			nil)
+		testutil.Ok(t, err)
+
+		// Note that we don't wait for the trace to be committed to storage.
+		// (If we were using a buffered Jaeger storage backend we would
+		// not give up on the first fetch attempt.)
+		response, err = client.Do(request)
+		testutil.Ok(t, err)
+		defer response.Body.Close()
+
+		body, err = ioutil.ReadAll(response.Body)
+		testutil.Ok(t, err)
+
+		bodyStr = string(body)
+		//nolint:lll
+		assertResponse(t, bodyStr, `{"result":{"resourceSpans":[{"resource":{"attributes":[{"key":"host.name","value":{"stringValue":"testHost"}}]},"instrumentationLibrarySpans":[{"instrumentationLibrary":{},"spans":[{"traceId":"W47/95gDgQPSabYzgT/GDA==","spanId":"7uGbfsPBsXM=","parentSpanId":"AAAAAAAAAAA=","name":"testSpan","kind":"SPAN_KIND_INTERNAL","startTimeUnixNano":"1544712660000000000","endTimeUnixNano":"1544712661000000000","attributes":[{"key":"attr1","value":{"intValue":"55"}},{"key":"internal.span.format","value":{"stringValue":"proto"}}]}]}]}]}}`)
+
+		testutil.Equals(t, http.StatusOK, response.StatusCode)
+
+		// Uncomment for interactive test
```

Comment:
would this happen in this same PR? I think if not, we could also remove the commented lines